srv/pillar/ceph/blacklist.sls

```diff
@@ -0,0 +1,17 @@
+# This is an example file to give you some hints on how the strucutre should look like
+# By adjusting thos file you can prevent the 'cephprocesses' module from failing if a
+# specified OSD is down. This can come in handy when you have a OSD/device that is known
+# the be failing but can't be removed yet because it needs invenstigation for whatever reason.
```

Comment:
. . . but can't be removed yet for whatever reason.

The phrase "because it needs investigation" is superfluous.